site/gatsby-site/page-creators/createBlogPages.js

```diff
@@ -0,0 +1,101 @@
+const path = require('path');
+
+const createBlogPages = async (graphql, createPage, { languages, reporter }) => {
+  const result = await graphql(
+    `
+      query BlogPosts {
+        allPrismicBlog(sort: { data: { date: DESC } }) {
+          edges {
+            node {
+              uid
+              lang
+              data {
+                metatitle
+                metadescription
+                slug
+                aitranslated
+                language
+                title {
+                  text
+                }
+                content {
+                  richText
+                  text
+                  html
+                }
+                image {
+                  url
+                  gatsbyImageData
+                }
+                date
+                author
+              }
+            }
+          }
+        }
+        oldBlogPosts: allFile(
```

Comment:
Should we eventually move all posts to Prismic? It will also help with keeping the codebase project agnostic.

Reply:
I think eventually yes but it might take a long time to do so. So for this first version we agreed with Sean to have old and new posts coexist. I can rename the variables though so it’s clearer